File: src/main/java/com/godwin/jsonparser/ui/tab/TabIdCollection.java
package com.godwin.jsonparser.ui.tab;

import java.util.LinkedList;
import java.util.List;

public class TabIdCollection {
    private static final List<Integer> LIST= new LinkedList<Integer>();
    private static final int MAX_COUNT=11;

    public int getId(){
        for (int i =0;i<MAX_COUNT;i++){
            if(!LIST.contains(i)) return i;
        }
        return MAX_COUNT;
    }
    public void removeId(){

    }
}

File: src/main/kotlin/com/godwin/jsonparser/ui/tabs/TabIdCollection.kt
package com.godwin.jsonparser.ui.tabs

class TabIdCollection {
    fun getId(): Int {
        for (i in 0 until MAX_COUNT) {
            if (i !in list) return i
        }
        return MAX_COUNT
    }

    fun removeId() {}

    companion object {
        private val list = mutableListOf<Int>()
        private const val MAX_COUNT = 11
    }
}
